regression/verilog/SVA/empty_sequence1.desc

```diff
@@ -0,0 +1,8 @@
+KNOWNBUG
+empty_sequence1.sv
+--bound 5
+^EXIT=10$
+^SIGNAL=0$
+--
+^warning: ignoring
+--
```

Comment:
Would you mind adding a note as to what the problem here is?

Reply:
Done